src/components/Home/candidate-section/candidate-section.tsx

```
@@ -0,0 +1,102 @@
+import SectionImage from '../../../assets/imgs/candidate-section.png';
+import MobileSectionImage from '../../../assets/imgs/candidate-section-mobile.png';
+import { LazyImage } from '../../Ui/lazy-images';
+
+import { useNavigate } from 'react-router';
+
+export const CandidateSection = ({
+}) => {
```

Biome (2.1.2)

[error] Unexpected empty object pattern.



(lint/correctness/noEmptyPattern)